src/mono/browser/runtime/package-lock.json

```diff
@@ -1,130 +1,203 @@
 {
   "name": "@microsoft/dotnet-runtime",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 3,
```

Comment:
Changes to this file should be reverted.

There is an upstream issue with using an old npm/whatever that makes it such that if you have a newer tool, it auto-upgrades the file, and you have to manually downgrade it, or do some other workaround.